src/shared/services/document-management/document-management.service.ts

```diff
@@ -26,4 +29,30 @@ export class DocumentManagementService {
         map(response => response.json())
       );
   }
+
+  createMediaViewer(caseField: CaseField): string {
+    let mediaViewer = {};
+    if (caseField.value) {
+      mediaViewer = {
+        document_binary_url: this.transformDocumentUrl(caseField.value.document_binary_url),
+        document_filename: caseField.value.document_filename,
+        content_type: this.getContentType(caseField),
+      }
+    }
+    return JSON.stringify(mediaViewer);
+  }
+
+  getContentType(caseField: CaseField): string {
+    let fileExtension = '';
+    if (caseField.value && caseField.value.document_filename) {
+      fileExtension = caseField.value.document_filename
+        .slice(caseField.value.document_filename.lastIndexOf('.') + 1);
+    }
+    return (fileExtension == 'pdf') ? DocumentManagementService.PDF : DocumentManagementService.IMAGE;
```

Comment:
Is it guaranteed that the type is image if it is not pdf? Couldn't it be a third type? This method seems to be problematic.

Reply:
Refactored by introducing an array with the list of images